Changelog.md

```diff
@@ -1,5 +1,13 @@
 ## Ignition Rendering
 
+### Ignition Rendering 2.6.0 (2020-10-05)
```

Comment:
Does this prompt a minor? Maybe it could be a patch release since there aren't new features?

Reply:
yeah patch release is better. updated to 2.5.1. 448e2169e7ea4a7bd3937e4d2d9c12de8414f97e